TicTacToe.py

```diff
@@ -87,7 +87,7 @@ def updateBoard(self, row, col, player_icon):
 		:param player_icon: the icon to be put in the space (traditionally X or O).
 		"""
 
-		if player_icon in (self.BLANK_POS_ICON, self.PLAYER_0_ICON, self.PLAYER_1_ICON):
+		if player_icon in self.PLAYER_0_ICON or self.PLAYER_1_ICON:
```

Comment:
I've never seen syntax like this, where you have an `in` with 2 options with an `or` in the middle. How exactly does this work?

Reply:
I could make it a tuple to make it more clear

```suggestion
		if player_icon in (self.PLAYER_0_ICON, self.PLAYER_1_ICON):
```

or simply add parentheses

```suggestion
		if player_icon in (self.PLAYER_0_ICON or self.PLAYER_1_ICON):
```

All of these are equivalent, I'm happy with any of them if you want to change it

Reply:
You can't have the `or` in the middle, `or` just evaluate the 2 variables against each other. As they are both strings (with how the code is currently setup) it will always evaluate as `self.PLAYER_0_ICON` (unless it is an empty string).

Your first suggestion is almost the same way it started, just minus `self.BLANK_POS_ICON`, which logically should never be sent to updateBoard (even more so in our current setup) but could possibly be needed in the future.